test/navigation.vader

```diff
@@ -462,3 +465,27 @@ Execute (edit routes):
 
 Expect (generic template):
   <?php
+
```

Comment:
Let's move these new tests to their own test file, `tests/goto.vader` (matching the autoload script name), since we're testing a different type of functionality.